src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let sum = days * 40;
+
+  if (sum < 120) {
+    return sum;
+  }
```

Comment:
Avoid using hardcoded values like `120` in the condition. Define constants with descriptive names to represent the thresholds and discounts.